### PR TITLE
Add travis and tox

### LINKED ISSUE #3: Add tox and travis
We should add travis and tox to ensure test pass on all the suported platforms (py27, py34, py35)

I'm wondering if we should also add coverage and pep8 checks

I can work on it

### PR DESCRIPTION
**Pull request by pokoli**
Closes #3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ref: https://docs.travis-ci.com/user/languages/python
+language: python
+env:
+  - TOXENV=py36
+  - TOXENV=py35
+  - TOXENV=py34
+  - TOXENV=py27
+  - TOXENV=update-pep8
```
Comment (mbohlool) [inline .travis.yml]: Did you add update-pep8 somewhere? I don't see it.

**Comment by mbohlool** [comment]
I tried to run tox, and got errors like `ImportError: No module named 'kubernetes'`. I don't think it is as straightforward as adding tox files to this repo. One thing do during test is to clone parent repo, update base submodule with the PR, and run tox on that.

**Comment by mbohlool** [comment]
I am giving this a try and will send another PR if you don't mind. will assign it to you.

**Comment by pokoli** [comment]
I'm closing this in favour of #9

**Pull request closed without merging**